sys/usb/usbus/dfu/dfu.c

```diff
@@ -240,12 +255,16 @@ static int _control_handler(usbus_t *usbus, usbus_handler_t *handler,
 {
     (void)usbus;
     (void)state;
+
     usbus_dfu_device_t *dfu = (usbus_dfu_device_t *)handler;
     DEBUG("DFU: Request: 0x%x\n", setup->request);
 
     /* Process DFU class request */
     if (setup->type & USB_SETUP_REQUEST_TYPE_CLASS) {
-        _dfu_class_control_req(usbus, dfu, setup);
+        if (_dfu_class_control_req(usbus, dfu, setup) < 0) {
+            DEBUG("DFU: control request failed\n");
```

Comment:
Small nitpick

```suggestion
            DEBUG("DFU: control request failed: %u\n", setup->request);
```

Makes the debug line a bit more useful when debugging

Reply:
Done